### PR TITLE
Implement Status Effect Magic Evasion modifiers

### PR DESCRIPTION
**Pull request by Xaver-DaRed**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

- Creates STATUS_MEVA family of modifiers that will be used in the upcoming rewrite of enfeebling spells.
- Corrects Bar-Effect spells potency, since they seemingly arent affected by enhancing skill.
- Substitutes STATUSRES with STATUS_MEVA when applicable. Hopefully I got them all.

## Steps to test these changes

For the time being? No steps.

### CODE REVIEW

File: scripts/globals/effects/asylum.lua [hunk 1]
```diff
@@ -6,42 +6,42 @@ require("scripts/globals/status")
 local effectObject = {}
 
 effectObject.onEffectGain = function(target, effect)
```
Comment (WinterSolstice8) [inline scripts/globals/effects/asylum.lua:8]: This is probably a massive nerf to how Asylum works, but I don't think we know the specifics either. ${STATUS}RES is probably _more_ accurate than pure meva. Thoughts?

File: scripts/globals/effects/perfect_defense.lua [hunk 2]
```diff
@@ -10,16 +10,16 @@ effectObject.onEffectGain = function(target, effect)
     target:addMod(xi.mod.UDMGBREATH, -effect:getPower())
     target:addMod(xi.mod.UDMGMAGIC, -effect:getPower())
     target:addMod(xi.mod.UDMGRANGE, -effect:getPower())
-    target:addMod(xi.mod.SLEEPRES, effect:getPower())
```
Comment (WinterSolstice8) [inline scripts/globals/effects/perfect_defense.lua:13]: This one's also a massive nerf to how it works like Asylum, but details aren't particularly known either.

**Comment by WinterSolstice8** [comment]
I know you had asked where "Resist!" procs come from but I legitimately forgot about bard songs

JP wiki says bard songs are resist proc 
http://wiki.ffo.jp/html/note16731.html

File: scripts/globals/effects/asylum.lua (hunk 1 above)
Comment (Xaver-DaRed) [reply scripts/globals/effects/asylum.lua:8]: Job points increase MEVA by 4 points each to asylum, so i would say it definately uses STATUS_MEVA modifier

The numbers, however can be increased. I didnt touch any values

**Comment by Xaver-DaRed** [comment]
Reverted bard song modifiers.

As for "Asylum" and "Perfect Defense", they do not trigger Resist! message, meaning, they probably give a massive boost to MEVA (specific to the status) akin to the MACC that Elemental Seal and such JAs give (256)

I would propose raising the potency of this modifiers instead.

Another note: STATUSRES is supposed to cap at 80%, meaning they would get nerfed even if left with STATUSRES.
And I have honest doubts they were having any actual effect anyway.

**Review by WinterSolstice8 — APPROVED** [review]

**Review by claywar — APPROVED** [review]

**Pull request merged**